bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifConnection.java

```diff
@@ -475,13 +493,28 @@ void parseXAddr(String message) {
     }
 
     private void parseDateAndTime(String message) {
+        Date openHABTime = new Date();
         String minute = Helper.fetchXML(message, "UTCDateTime", "Minute>");
         String hour = Helper.fetchXML(message, "UTCDateTime", "Hour>");
         String second = Helper.fetchXML(message, "UTCDateTime", "Second>");
         String day = Helper.fetchXML(message, "UTCDateTime", "Day>");
         String month = Helper.fetchXML(message, "UTCDateTime", "Month>");
         String year = Helper.fetchXML(message, "UTCDateTime", "Year>");
-        logger.debug("Camera  UTC dateTime is: {}-{}-{}T{}:{}:{}", year, month, day, hour, minute, second);
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+        try {
+            String time = year + "-" + month + "-" + day + "T" + hour + ":" + minute + ":" + second;
```

Comment:
Double check; XML has leading zero's for values below 10 ? Otherwise this string does not match the `dateFormat`

Reply:
Thank you, I made the changes needed and tested it which confirmed you were right.